docs/en/e-service-pdnd.rst

```diff
@@ -394,7 +394,7 @@ The PDND Authorization Server MUST also validate the ``client_assertion`` JWT as
 
     Header:
 
-        - Ensure that the ``typ`` claim is present and that its value is ``jwt``.
+        - Ensure that the ``typ`` claim is present and that its value is ``JWT``.
```

Comment:
https://github.com/italia/eudi-wallet-it-docs/pull/541/files/b1726220c199856fd975f9c418aafc020f27ee33..8e5abfbafddb3d684b8d90fe1fa7f85b157240b7#r1952782229